components/brave_rewards/resources/extension/brave_rewards/actions/rewards_panel_actions.ts

```diff
@@ -9,11 +9,9 @@ import { types } from '../constants/rewards_panel_types'
 
 export const createWallet = () => action(types.CREATE_WALLET, {})
 
-export const onWalletCreated = () => action(types.ON_WALLET_CREATED, {})
-
-export const onWalletCreateFailed = () => action(types.ON_WALLET_CREATE_FAILED, {})
-
-export const onWalletCorrupted = () => action(types.ON_WALLET_CORRUPTED)
+export const onWalletInitialized = (result: number) => action(types.ON_WALLET_INITIALIZED, {
```

Comment:
instead of number, you can use `Rewards.Result`

Reply:
Done